Formula/alpine.rb

```diff
@@ -11,8 +11,15 @@ class Alpine < Formula
     sha256 "8f52d4ebe9e445ec975cabdd74c4a48cef80eebb21f18c33644e99de1a6d2173" => :mountain_lion
   end
 
+  option "with-maildir", "Compile with support for Maildir format mailboxes"
+
   depends_on "openssl"
 
+  patch do
+    url "http://patches.freeiz.com/alpine/patches/alpine-2.20/maildir.patch.gz"
```

Comment:
It's provided by them, so no, they already have it. They provide build options as patches. I realize that's not the way you want the world to work, so clearly we're at an impasse here. Thanks!